project/backend/src/main/java/com/google/sps/data/RestaurantScorer.java

```diff
@@ -115,7 +115,7 @@ private static long roundMetersToMiles(double numMeters) {
   /** Skews ratings so that ratings with a lower number of ratings are weighed less. */
   private static double calculateRatingScore(
       double avgRating, int numRatings, DescriptiveStatistics statistics) {
-    int numInitialRatings = (int) Math.round(statistics.getPercentile(25));
+    int numInitialRatings = (int) Math.round(statistics.getPercentile(10));
```

Comment:
is this change related ?

Reply:
Oh no it's not, probably should've put this in a different PR